### DISCUSSION TITLE
blog/package-git-into-ghost-docker-image

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#3] [Announcements]
# blog/package-git-into-ghost-docker-image

这篇文章介绍如何将Git环境封装到Docker中，以避免Git环境被销毁的问题，以更好地用Github备份Ghost博客。

https://rercel.com/blog/package-git-into-ghost-docker-image

**Comment by Kevin-free** [comment]
感谢分享！我在实践过程中想反馈几点问题：
1. 需要`chmod +x docker-entrypoint.sh`，否则 fly deploy 报错了。
2. 是否可以将`git config --global --add safe.directory /var/lib/ghost/content` 等 git 的初始化设置放到`Dockerfile`或者哪默认执行？
3. 为什么只备份`/var/lib/ghost/content`目录，可以备份`/var/lib/ghost`目录吗？在Mac本地调式？
